windows/utils/win_get_set_disk_online.yml

```diff
@@ -51,11 +51,5 @@
     - win_powershell_cmd_output.stdout_lines | length != 0
     - win_powershell_cmd_output.stdout_lines[0] | lower == 'online'
 
-- name: "Check disk online status"
```

Comment:
The assert task can be moved after Print task.

Reply:
Removed 'assert' since here just getting the disk status.